pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 [project]
 name = "one_py_sdk"
-version = "1.2.0"
+version = "1.3.3"
```

Comment:
If you're just adding new functionality, shouldn't this change from 1.2.0 to 1.3.0?

Reply:
I'll bump the version to 1.4.0. There's already a 1.3.2 out there, and somehow the version bump got lost in the shuffle and didn't get merged to master.